data/json/monsters/feral_humans.json

```diff
@@ -160,13 +160,38 @@
   {
     "id": "mon_feral_human_axe",
     "type": "MONSTER",
-    "description": "Their pupils are dilated and what can be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat them like one of their own.",
-    "copy-from": "mon_feral_human_pipe",
-    "melee_dice": 2,
+    "name": { "str": "deranged axeman", "str_pl": "deranged axemen" },
+    "description": "They stumble along dragging a bloodstained fire axe.  Their face is twisted into a sadistic grin, and they walk among the dead as though already one of them.",
```

Comment:
``fire axe``
*angry #58014 noises*

Reply:
Oops, didn't notice that PR. Easy fix at least, I'll just change it so the type of axe is unspecified.